tools/workspace-plugin/src/plugins/workspace-plugin.ts

```diff
@@ -555,14 +556,19 @@ function buildReactIntegrationTesterProjectConfiguration(
   ) {
     return {};
   }
+
   // react v9: apply to libraries and stories projects
-  const isStoriesProject = config.tags.includes('type:stories');
-  if (!config.tags.includes('vNext') || (config.projectJSON.projectType !== 'library' && !isStoriesProject)) {
+  const isv8Lib = config.tags.includes('v8');
+  const isv9Lib = config.tags.includes('vNext');
+  const isV9StoriesProject = isv9Lib && config.tags.includes('type:stories');
+  const isTool = config.tags.includes('tools');
+
+  if (isTool || !(isv9Lib || isV9StoriesProject || isv8Lib)) {
```

Comment:
`rit` test targets will now work for UI v8 projects if enabled within nx.json config   ( tools are ignored )